vnext/Shared/Networking/OriginPolicyHttpFilter.cpp

```diff
@@ -511,14 +511,14 @@ void OriginPolicyHttpFilter::ValidatePreflightResponse(
 
   auto controlValues = ExtractAccessControlValues(response.Headers());
 
-  auto iRequestArgs = request.Properties().Lookup(L"RequestArgs");
+  auto props = request.Properties();
   // Check if the origin is allowed in conjuction with the withCredentials flag
   // CORS preflight should always exclude credentials although the subsequent CORS request may include credentials.
-  ValidateAllowOrigin(controlValues.AllowedOrigin, controlValues.AllowedCredentials, iRequestArgs);
+  ValidateAllowOrigin(controlValues.AllowedOrigin, controlValues.AllowedCredentials, props);
 
   // See https://fetch.spec.whatwg.org/#cors-preflight-fetch, section 4.8.7.5
   // Check if the request method is allowed
-  bool withCredentials = iRequestArgs.as<RequestArgs>()->WithCredentials;
+  bool withCredentials = props.Lookup(L"RequestArgs").as<RequestArgs>()->WithCredentials;
```

Comment:
>rops.Lookup(L"Re [](http://example.com/codeflow?start=26&length=16)

Nit: RequestArgs is already looked up on props earlier... in Line 450

Reply:
please ignore, is in other function, indentation style used for functions makes it hard to read when a new scope is introduced....